opa/stackable/bin/process-logs

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
```

Comment:
I think we should run scripts in strict mode, eg:

```suggestion
#!/usr/bin/env bash
set -euo pipefail
```

Reply:
`-eo` Done in b5533dbf5b94a8ae8b52bdc43a36a2bc44fcf3bf

In this case, the script is used in a redirect, eg: `opa run ... &> >(process-logs)`. So any failure will just stop the log processing and all output from `opa` will be lost (but the process will continue running).